tools/postinstall-android.js

```diff
@@ -74,6 +74,7 @@ replaceTextInFile(path.join(appProjectRoot, 'MobileSync', 'build.gradle'), oldSm
 replaceTextInFile(path.join(appProjectRoot, 'SalesforceHybrid', 'build.gradle'), oldMobileSyncDep, 'api project\(\':MobileSync\'\)');
 
 console.log('Fixing root level Gradle file for the generated app');
+replaceTextInFile(path.join(appProjectRoot, 'settings.gradle'), "include \":CordovaLib\"", "");
```

Comment:
This was the problem - it has 2 references to `CordovaLib`, the one that is implicitly pulled in through `SalesforceHybrid` and the one coming from the generated project.